snippets/src/lib.rs

```diff
@@ -430,6 +430,12 @@ pub struct Snippet<L> {
     language: PhantomData<L>,
 }
 
+impl<L> PartialOrd for Snippet<L> {
```

Comment:
Sanity check: I'm assuming that this was just a quick fix that was intentionally added.

Reply:
Yep, it was a lint issue